test/lib/next-test-utils.ts

```diff
@@ -800,7 +800,7 @@ export async function retry<T>(
   fn: () => T | Promise<T>,
   duration: number = 3000,
   interval: number = 500,
-  description?: string
+  description: string = fn.name
```

Comment:
Nicer to just pass a named function to `retry` IMO which also helps in callstacks.

These should really be options 🤷🏻